org.salutatoi.Cagou.appdata.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version='1.0' encoding='utf-8'?>
```

Comment:
Desktop and appdata files ideally should be part of upstream repositories and installed by project buildsystem if possible – other distributions are also using them.

Reply:
Indeed. Next version will be Python 3 only and I've done many changes already, I'll add those script upstream for the next version, but I rather keep them here for this 0.7.0 version.